src/components/select/demoTrackBy/script.js

```diff
@@ -0,0 +1,23 @@
+angular.module('selectDemoTrackBy', ['ngMaterial', 'ngMessages'])
```

Comment:
Sorry, I forgot to add this to the last review, can you please wrap this in an IIFE like the demoBasicUsage?